src/styles/blocks/catalog.scss

```diff
@@ -18,8 +18,15 @@
     border-radius: 5px;
     margin-bottom: 48px;
 
+    transition: $duration;
```

Comment:
use transition property for the styles you need
https://developer.mozilla.org/en-US/docs/Web/CSS/transition-property